TykTechnologies/tyk
# Fix bundle directory creation

This was solved by #1111.
To test this:
- Create a plugin bundle.
- Setup an API definition with the new bundle name.
- Make sure that the middleware bundle directory doesn't exist (usually `/opt/tyk-gateway/middleware/bundles`).
- Start Tyk
- The bundle directory should be available and the bundle loader shouldn't fail.
